packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -333,14 +333,16 @@ class DefaultTabController extends StatefulWidget {
   ///
   /// {@macro flutter.widgets.child}
   final Widget child;
-
+  
```

Comment:
I would reformat it in the following manner:

```
  /// The closest instance of this class that encloses the given context.
  ///
  /// {@tool sample}
  /// Typical usage is as follows: 
  /// ```dart
  /// TabController controller = DefaultTabController.of(context);
  /// ```
  /// {@end-tool}
```

Reply:
That way, the first sentence of the API doc will not be part of the code sample.